ruby-challenges.rb

```diff
@@ -10,6 +10,17 @@
 num3 = 221
 # Expected output: '221 is odd'
 
+def num_determine number
```

Comment:
great structure. Woohoo you remembered to print the function call with an argument